mirage/factories/node.ts

```diff
@@ -134,6 +137,23 @@ export default Factory.extend<MirageNode & NodeTraits>({
             node.save();
         },
     }),
+
+    withAffiliatedInstitutions: trait({
```

Comment:
Please type the trait, (as other traits above) and remove `any`s for `node` and `server`.